storage/objects/objects_test.go

```diff
@@ -204,6 +204,26 @@ func TestObjects(t *testing.T) {
 		}
 	})
 
+	t.Run("downloadByteRange", func(t *testing.T) {
+		dir, err := ioutil.TempDir("", "downloadByteRangeTestTempDir")
```

Comment:
You could potentially de-duplicate this with your other sample test and just create one tempdir at the beginning of TestObjects to use for both subtests-- especially if you're going to add additional samples that will require a local dir.